travis/check_tags.py

```diff
@@ -0,0 +1,74 @@
+            real_errors.update(errors_commit)
+    return real_errors
+
+
+def handler_commit(commit, is_addons_dev, symbol_in_branch, version):
+    development_tags = [':memo:', ':fire:', ':fire_engine:', ':tv:', ':lock:', ':bath:', ':green_heart:', ':cat:', ':bomb:']
+    release_tags = [':tada:', ':zap:', ':sparkles:', ':rainbow:', ':ambulance:', ':heart_eyes:', ':cherries:', ':book:', ':euro:', ':handshake:', ':shield:', ':arrow_up:', ':arrow_down:', ':x:', ':sos:', ':peace_symbol:']
+    verion_tags_dict = {'zero': '0', 'one': '1', 'two': '2', 'three': '3', 'four': '4', 'five': '5', 'six': '6', 'seven': '7', 'eight': '8', 'nine': '9'}
+    verion_tags = [':zero:', ':one:', ':two:', ':three:', ':four:', ':five:', ':six:', ':seven:', ':eight:', ':nine:']
+    requirements_tags_of_version = [':x:', ':arrow_up:', ':arrow_down:', ':sos:', ':tada:']
+
+    errors_commit = {}
+    match_tags_commit = re.search(r'^(:.+:) \w+', commit).group(1)
+    if not match_tags_commit:
+        error = {commit: 'There are no tags in the commit!'}
+        errors_commit.update(error)
+        return errors_commit
+    list_tags = re.findall(r'(:\w+:)', match_tags_commit)
+    if is_addons_dev or symbol_in_branch and not any(tag in list_tags for tag in development_tags):
+        error = {commit: 'For dev branches must be Development tags!'}
+        errors_commit.update(error)
+        return errors_commit
+    elif any(tag in list_tags for tag in requirements_tags_of_version):
+        if not any(tag in list_tags for tag in verion_tags):
+            error = {commit: 'Must be Version tags!'}
+            errors_commit.update(error)
+            return errors_commit
+        list_digits = [x for x in match_tags_commit.split(':') if x in verion_tags_dict.keys()]
+        version_in_commit = ''
+        for digit in list_digits:
+            version_in_commit += verion_tags_dict.get(digit)
+        if version.replace(".", "") != version_in_commit:
+            error = {commit: 'Version in commit is wrong!'}
+            errors_commit.update(error)
+            return errors_commit
+        # list of indices (requirements of version's tags in commit)
+        index_requirements_tags_version = [list_tags.index(i) for i in list_tags if i in requirements_tags_of_version]
+        # list of indices (version's tags in commit)
+        index_verion_tags = [list_tags.index(i) for i in list_tags if i in verion_tags]
+        # Check proper order of tags in commit: comparison indices "requirements of versions tags" and "versions tags"
+        if not index_requirements_tags_version[-1] < index_verion_tags[0]:
+            error = {commit: 'Proper order tags in commit is wrong!'}
+            errors_commit.update(error)
+            return errors_commit
+    else:
+        if not any(tag in list_tags for tag in release_tags):
+            error = {commit: 'For stable branches must be Release tags!'}
```

Comment:
when it's dev branch + dev tags , you script will say ``For stable branches must be Release tags``